jerry-core/vm/vm.c

```diff
@@ -184,7 +184,7 @@ vm_op_set_value (ecma_value_t object, /**< base object */
 /**
  * Decode table for both opcodes and extended opcodes.
  */
-static const uint16_t vm_decode_table[] =
+static const uint16_t vm_decode_table[] STORE_IN_ROM =
```

Comment:
I would expect that just the table is aligned. If there is a way to check this I can create a small test and tell you for sure.